scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -231,15 +231,29 @@
       <ul class="list-group">
         <li class="list-group-item">
           <a href="https://sift.bii.a-star.edu.sg/www/SIFT_help.html" target="_blank">SIFT</a>
-          <span class="float-right">{{ variant.sift_predictions|join(', ') }}</span>
+          {% if variant.sift_predictions  %}
```

Comment:
Works nicely! I won't add screenshots since they look like your example on the PR description!